net-core/Ical.Net/Ical.Net.Collections/Ical.Net.Collections.csproj

```diff
@@ -1,6 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
-    <TargetFramework>netstandard1.4</TargetFramework>
+    <TargetFrameworks>netstandard1.3;net46</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputType>library</OutputType>
```

Comment:
Yup you can do that however bear in mind the default behaviour when you run `dotnet pack` is to list the dependencies as separate packages so you'd need to push the collections etc libraries separately. That is one large change compared to full .net fwk stuff. There is a way around it by including dlls in the nupkg like you currently do with the nuspec but i'm not sure its advised.